dummy/Cargo.toml

```diff
@@ -2,15 +2,14 @@
 name = "dummy"
 version = "0.1.0"
 authors = [
-    "Dzmitry Malyshau <kvark@mozilla.com>",
+    "Dzmitry Malyshau <kvark@mozilla.com>"
```

Comment:
We can make these inherent both version and authors

Reply:
Does the `version` of deno_webgpu also inherit the same field values?

Reply:
No, deno-webgpu is kinda a separate project but resides in repo so we can test with it